asmcli/components/ca/managed-cas.sh

```diff
@@ -75,6 +75,59 @@ x_enable_workload_certificate_on_membership() {
      -d "${ENABLEFEATURE}" "https://${GKEHUB_API}/v1alpha/projects/${FLEET_ID}/locations/global/features/workloadcertificate?update_mask=membership_specs"
 }
 
+x_wait_for_gke_hub_api_enablement() {
+  info "Wait 3 minutes for GKE HUB API enablement to propagate to the systems ..."
```

Comment:
waIt 3 minutes is it enough?

Reply:
For the production environment, 3 minutes should be enough. For other environments, there may be some issues with the recent implementation, which leads to a long propagation latency.

Reply:
think it should be?..this isn't for the cluster restart though...

Reply:
This isn't for the cluster restart. The GKE hub API enablement propagation latency for non-production environments can be tens of minutes, which seems to be a bug. The wait here is for normal propagation delays and 3-minute is a reasonable value.

Reply:
Could we check ```gcloud alpha container fleet workload-certificate describe``` to make sure the provision google ca is enabled?

Reply:
The gcloud alpha call is equivalent to the curl call and the curl call does not require the installation of an alpha version of gcloud.